libraries/libfc/include/fc/crypto/bls_signature.hpp

```diff
@@ -8,83 +8,39 @@
 #include <fc/reflect/variant.hpp>
 #include <bls12-381/bls12-381.hpp>
 
-
-namespace fc { namespace crypto { namespace blslib {
-
-   using namespace std;
+namespace fc::crypto::blslib {
 
    namespace config {
-      constexpr const char* bls_signature_base_prefix = "SIG";
-      constexpr const char* bls_signature_prefix = "BLS";
+      constexpr std::string_view bls_signature_prefix = "SIG_BLS_";

```

Comment:
```suggestion
      constexpr std::string bls_signature_prefix = "SIG_BLS_";
```
better to make it a string (like for the key prefixes) and avoid creating a string every time `to_string` is called.